Cargo.lock

```diff
@@ -1,27 +1,47 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
Try not to update cargo.lock since it's not necessary?

Reply:
This project depend ckb-script, the ckb-script depend ckb-log, the ckb-log depend sentry, the sentry depend reqwest, the reqwest is a higher-level HTTP client.